R/test-elements.R

```diff
@@ -993,6 +993,14 @@ volume_calibration_page <- function(url, type = tools::file_ext(url),
     ),
     shiny::p("")
   )
+  audio_NAFC_page(label = "volume_calibration",
```

Comment:
Inserts some space between text and button. Maybe better use some CSS here?

Reply:
No, I mean that this function call doesn't do anything, because it's not the final expression in the function. `audio_NAFC_page` is called twice which seems like it must be a mistake?